findlibs/__init__.py

```diff
@@ -34,16 +44,21 @@ def find(name):
 
     for root in roots:
         for lib in ("lib", "lib64"):
-            fullname = os.path.join(root, lib, "lib{}{}".format(name, extension))
+            fullname = os.path.join(root, lib, "lib{}{}".format(lib_name, extension))
             if os.path.exists(fullname):
                 return fullname
 
-    for what in ("HOME", "DIR"):
-        LIB_HOME = "{}_{}".format(name.upper(), what)
-        if LIB_HOME in os.environ:
-            home = os.path.expanduser(os.environ[LIB_HOME])
+    env_prefixes = [pkg_name.upper(), pkg_name.lower()]
```

Comment:
This is doing a bit more than the PR name/description would suggest. But, I certainly welcome the feature, and it is suitable with the supported modules and platforms at the centre...

Reply:
That said, can you add to the PR title/description this functionality (that it checks for both prefixing lower/uppercase versions of the package name)?